public/tutorials/using-infura-custom-provider.md

```diff
@@ -1,136 +1,152 @@
+
+     ```javascript
+     provider = new HDWalletProvider(mnemonic, "https://ropsten.infura.io/<Infura_Access_Token>", 2);
+     ```
+
+     (Recall that the index is zero-based, so `2` is the third address.)
+
+## Use an ether faucet
+
+Make sure you have enough ether in your account to do the deployment. You can acquire ether on the Ropsten network through a service known as a faucet. While there are multiple faucet sites out there, one service we recommend is hosted on [EthTools](https://ethtools.com/).
+
+1. Navigate to the [EthTools Ether Faucet](https://ethtools.com/ropsten/tools/faucet/).
+
+1. Enter your mnemonic, and select how much ether you would like (maximum of 5).
+
+1. The faucet will link to your first account. Click "Request Ether" to submit your request.
+
+1. Within a short period of time, your account will be populated with the requested ether.
+
+   <p class="alert alert-info">
+   <strong>Note</strong>: You can also request Ether through [MetaMask](https://metamask.io). Connect to your account on Ropsten, and click the "Buy" button, which will give you a link to MetaMask's Ropsten Test Faucet, which works similarly to above.
+   </p>
+
+We are now ready to deploy to Ropsten!
+
+## Deploy the contract
+
+1. Compile your project, if not already done:
+
+   ```shell
+   truffle compile
+   ```
+
+1. Deploy to the Ropsten network:
+
+   ```shell
+   truffle migrate --network ropsten
+   ```
+
+   If all goes well, you should see a response that looks similar to the following:
+
+   ```shell
+   Using network 'ropsten'.
+
+   Running migration: 1_initial_migration.js
+     Deploying Migrations...
+     ... 0xd79bc3c5a7d338a7f85db9f86febbee738ebdec9494f49bda8f9f4c90b649db7
+     Migrations: 0x0c6c4fc8831755595eda4b5724a61ff989e2f8b9
+   Saving successful migration to network...
+     ... 0xc37320561d0004dc149ea42d839375c3fc53752bae5776e4e7543ad16c1b06f0
+   Saving artifacts...
+   Running migration: 2_deploy_contracts.js
+     Deploying MyContract...
+     ... 0x7efbb3e4f028aa8834d0078293e0db7ff8aff88e72f33960fc806a618a6ce4d3
+     MyContract: 0xda05d7bfa5b6af7feab7bd156e812b4e564ef2b1
+   Saving successful migration to network...
+     ... 0x6257dd237eb8b120c8038b066e257baee03b9c447c3ba43f843d1856de1fe132
+   Saving artifacts...
+   ```
+
+   Your transaction IDs will be different though.
+
+   <p class="alert alert-info">
+   <strong>Note</strong>: If you receive an error `Error: Exceeds block gas limit
+`, you may need to manually set the gas limit for your contract. See the [Truffle Configuration](/docs/advanced/configuration) for details.
+   </p>
+
+1. If you want to verify that your contract was deployed successfully, you can check on [Etherscan](https://etherscan.io). In the search field, type in the transaction ID for your contract. In the above example, the transaction ID is:
```

Comment:
You should like to the Roptsen Etherscan here, or at least call out that they should navigate to the ropsten section of Etherscan. I could be wrong, but if the Ropsten tx is searched for in the main section of Etherscan, it won't be found, potentially causing confusion even if the transaction succeeded.